### PR TITLE
Project: Design Teardown

### PR DESCRIPTION
**Pull request by edxco**
For this project we done:

Building a heatmap of the Smashing magazine website
Align all elements with float and flex or grid.
Use article tag for all articles listed on the page.
Build your layout using a width of 1440px on the original page.

### CODE REVIEW

**Comment by edxco** [comment]
Working on Stylelint errors

**Comment by edxco** [comment]
After fixing linter errors

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+  align-items: baseline;
+  width: 200px;
+}
+
+.main_pics {
+  height: 100%;
+}
+
+.search_var {
+  width: 200px !important;
```
Comment (MuriloRoque) [inline css/style.css]: Please, remove the !important rule here, it's considered bad practice. You should achieve this result without needing to use this.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+}
+
+/* Header */
+.header {
+  padding-left: 0;
+  justify-content: space-around;
+}
+
+.header li {
+  border-radius: 10px;
```
Comment (MuriloRoque) [inline css/style.css]: You are using this property 7 times in your file, it would be better to create a class for it, agree?

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @edxco and @Nexch :wave: 

### Your page is very nicely implemented, you got the main idea of this project :+1: We just need to fix some issues, shall we?

## Page Structure

- [x] Make sure to build your layout using a width of **1440px** on the original page. You can use the Developer Tools to match that screen size. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/pageLayoutScreenSize.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/))

**Note:** Most of the things I will point out here might have happened because of the above requirement.

- [ ] We have a horizontal scroll in your page, and we need to fix this, please try playing with the width of your elements to fit them all inside of the page's width.

### 2nd row

- [x] Your page currently has 4 articles in this row, while in the original there are four, please check below:

***YOURS:***

Screenshot from 2020-08-27 15-07-41
<br>

***ORIGINAL:***

Screenshot from 2020-08-27 15-07-51
<br>

- [x] The last part before the footer also has some differences, yours has 3 columns, but the original has 2, check below:

***YOURS:***

Screenshot from 2020-08-27 15-19-25

<br>

***ORIGINAL:***

Screenshot from 2020-08-27 15-19-29
<br>

## General

- [x] Please check my comments in your code below.

## Summary

- Your code is very good, semantic HTML and overall good practices in your CSS :+1: 
- Your repository is professional, I really liked it :smiley: 
- Your page layout is obeying most of the requirements, we just need some adjustments to move forward :rocket: 

## Kindly make the requested changes and submit for another review, please :pray: 

## I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by edxco** [comment]
Review error

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

This Project is to match exactly the same layout as the website, yours needs a little bit more tweaking

* [ ] Files not related to the project should not be committed and added to .gitignore (e.g. `package-lock.json`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- [x] The Search button should be larger

- [x] "don't miss these latest articles" is not position well

- [x] the icon boxes are not positioned well 

image

- [x]  The 3 Rows Articles should take a third of the page by each row, without the extra space

image

- [x] the footer should be larger in height and with more spacing between links 

image

_work more on matching the spacings and positions just like the website shows on a 1440px width_

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by SpaYco** [comment]
P.S: you don't really need to close the PR every time you make a change, there's no problem in doing it though, so whatever suits you!

**Review by kevcap — APPROVED** [review]
# [STATUS] PROJECT APPROVED 🍾

## Congratulations team! Good job making my previously requested changes. You have my approval to go ahead and merge this pull request!




## As always, I wish you all the very best and see you both at the top🚀

⚠  Warning ⤵
- [ ] Guys, next time take care with the way you write some commits, they should be descriptive so the other programmer can see exactly what you changed with that commit. [Check this link](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
image



👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Pull request merged**